app/src/main/res/values/styles.xml

```diff
@@ -89,6 +89,11 @@
 
     </style>
 
+    <style name="AppTheme.CustomMenuText" parent="AppTheme">
+        <item name="android:actionMenuTextAppearance">@style/H3</item>
+        <item name="android:actionMenuTextColor">?attr/progressive_color</item>
+    </style>
```

Comment:
We have already used a `MaterialToolbar` in the fragment, would it be possible to add a `TextView` instead of applying a style on a menu item?